ui/raidboss/data/02-arr/raid/t7.txt

```diff
@@ -4,274 +4,274 @@
-127.8 "Tail Slap" sync /:Melusine:7A8:/
-130.8 "Circle Of Flames x2" #sync /:Melusine:7AA:/
-135.1 "Cursed Voice" sync /:Melusine:7AC:/
-145.3 "Circle Blade" sync /:Melusine:7AB:/
-147.3 "Tail Slap" sync /:Melusine:7A8:/
-148.4 "Circle Of Flames x2" #sync /:Melusine:7AA:/
-159.2 "Cursed Voice" sync /:Melusine:7AC:/
-163.1 "Tail Slap" sync /:Melusine:7A8:/
-164.2 "Circle Of Flames x2" #sync /:Melusine:7AA:/
-169.1 "Circle Blade" sync /:Melusine:7AB:/
-179.1 "Tail Slap" sync /:Melusine:7A8:/
-180.2 "Circle Of Flames x2" #sync /:Melusine:7AA:/
-184.5 "Cursed Voice" sync /:Melusine:7AC:/
-193.3 "Circle Blade" sync /:Melusine:7AB:/
-195.2 "Tail Slap" sync /:Melusine:7A8:/
-196.6 "Circle Of Flames x2" #sync /:Melusine:7AA:/
+0.0 "--sync--" sync / 00:0839::Bioweapon Storage will be sealed off/ window 10,10
+6.4 "Tail Slap" sync / 1[56]:Melusine:7A8:/ window 7,0
+12.3 "Cursed Voice" sync / 1[56]:Melusine:7AC:/
+14.3 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
+24.8 "Circle Blade" sync / 1[56]:Melusine:7AB:/
+26.5 "Tail Slap" sync / 1[56]:Melusine:7A8:/
+30.3 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
+36.4 "Cursed Voice" sync / 1[56]:Melusine:7AC:/
+42.5 "Tail Slap" sync / 1[56]:Melusine:7A8:/
+48.8 "Circle Blade" sync / 1[56]:Melusine:7AB:/
+50.5 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
+60.4 "Cursed Voice" sync / 1[56]:Melusine:7AC:/
+61.5 "Tail Slap" sync / 1[56]:Melusine:7A8:/
+66.3 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
+72.7 "Circle Blade" sync / 1[56]:Melusine:7AB:/
+77.5 "Tail Slap" sync / 1[56]:Melusine:7A8:/
+82.3 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
+86.6 "Cursed Voice" sync / 1[56]:Melusine:7AC:/
+93.6 "Tail Slap" sync / 1[56]:Melusine:7A8:/
+97.1 "Circle Blade" sync / 1[56]:Melusine:7AB:/
+99.1 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
+110.7 "Cursed Voice" sync / 1[56]:Melusine:7AC:/
+111.8 "Tail Slap" sync / 1[56]:Melusine:7A8:/
+114.8 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
+121.2 "Circle Blade" sync / 1[56]:Melusine:7AB:/
+127.8 "Tail Slap" sync / 1[56]:Melusine:7A8:/
+130.8 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
+135.1 "Cursed Voice" sync / 1[56]:Melusine:7AC:/
+145.3 "Circle Blade" sync / 1[56]:Melusine:7AB:/
+147.3 "Tail Slap" sync / 1[56]:Melusine:7A8:/
+148.4 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
+159.2 "Cursed Voice" sync / 1[56]:Melusine:7AC:/
+163.1 "Tail Slap" sync / 1[56]:Melusine:7A8:/
+164.2 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
+169.1 "Circle Blade" sync / 1[56]:Melusine:7AB:/
+179.1 "Tail Slap" sync / 1[56]:Melusine:7A8:/
+180.2 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
+184.5 "Cursed Voice" sync / 1[56]:Melusine:7AC:/
+193.3 "Circle Blade" sync / 1[56]:Melusine:7AB:/
+195.2 "Tail Slap" sync / 1[56]:Melusine:7A8:/
+196.6 "Circle Of Flames x2" #sync / 1[56]:Melusine:7AA:/
 
 ### Phase 2 (80%): shriek, 3 adds 30 seconds apart
 300.0 "--sync--" sync /:Melusine HP at 79%/ window 300,0
```

Comment:
For what it's worth, `0x0D` hp lines are removed in the alpha.  So, this (somehow?) needs a bigger sync, maybe on the first shriek and lotus blade lines?

Reply:
Yes, this will get handled with the manual changes. Would you prefer those to be in a completely separate PR, or is a separate commit here good enough for those?

Reply:
I think maybe better to do manual changes and automated changes in separate PRs.  If you make any changes to the automation, then it'll be easier if manual changes aren't interspersed.

Reply:
I think maybe better to do manual changes and automated changes in separate PRs.  If you make any changes to the automation, then it'll be easier if manual changes aren't interspersed.

Reply:
Makes sense. I'll keep the list of known issues then and fix them all after we get the automatic conversions done.